test/e2e/tests/errors.spec.js

```diff
@@ -163,13 +455,15 @@ describe('Sentry errors', function () {
           await driver.navigate();
           await driver.fill('#password', 'correct horse battery staple');
           await driver.press('#password', driver.Key.ENTER);
+
           // Trigger error
-          driver.executeScript('window.stateHooks.throwTestError()');
+          await driver.executeScript('window.stateHooks.throwTestError()');
+
           // Wait for Sentry request
           await driver.wait(async () => {
             const isPending = await mockedEndpoint.isPending();
             return isPending === false;
-          }, 10000);
+          }, 3000);
```

Comment:
This was made 3000 to match the other test. If it takes longer than 3 seconds here for the request to arrive, then it implies the test where we're expecting it to never arrive isn't waiting long enough.